drivers/mtd/smart.c

```diff
@@ -856,7 +856,8 @@ static ssize_t smart_reload(struct smart_struct_s *dev, FAR uint8_t *buffer,
 
   /* Read the full erase block into the buffer */
 
-  finfo("Read %d blocks starting at block %d\n", mtdblocks, mtdstartblock);
+  finfo("Read %" PRIdOFF "blocks starting at block %" PRIdOFF "\n",
```

Comment:
```suggestion
  finfo("Read %" PRIdOFF " blocks starting at block %" PRIdOFF "\n",
```
Missing whitespace after the offset value.

Reply:
Nice catch! Thank you!